wallet/src/test/java/io/neow3j/wallet/WalletTest.java

```diff
@@ -0,0 +1,146 @@
+package io.neow3j.wallet;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.neow3j.crypto.Keys;
+import io.neow3j.crypto.NEP2;
+import io.neow3j.crypto.exceptions.CipherException;
+import io.neow3j.wallet.nep6.NEP6Account;
+import io.neow3j.wallet.nep6.NEP6Wallet;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.io.IOException;
+import java.net.URL;
+import java.security.InvalidAlgorithmParameterException;
+import java.security.NoSuchAlgorithmException;
+import java.security.NoSuchProviderException;
+import java.util.Collections;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class WalletTest {
+
+    @Rule
+    public ExpectedException exceptionRule = ExpectedException.none();
+
+    @Test
+    public void testCreateDefaultWallet() {
+        Wallet w = new Wallet.Builder().build();
+        assertEquals(w.getName(), "neow3jWallet");
+        assertEquals(w.getVersion(), Wallet.CURRENT_VERSION);
+        assertTrue(w.getAccounts().isEmpty());
+    }
+
+    @Test
+    public void testCreateWalletFromNEP6File() throws IOException {
+
+        URL nep6WalletFile = Thread.currentThread().getContextClassLoader().getResource("wallet.json");
+        ObjectMapper mapper = new ObjectMapper();
+        NEP6Wallet nep6Wallet = mapper.readValue(nep6WalletFile, NEP6Wallet.class);
+
+        Wallet w = Wallet.fromNEP6Wallet(nep6Wallet).build();
```

Comment:
As discussed, create a method called "load" or something like this, that will get as parameter a String or Path with the path to the wallet (.json).